core/src/tests/parachain.rs

```diff
@@ -124,9 +124,6 @@ fn cidp_and_dp_relay(
 			let parent_header = client.header(parent).unwrap().unwrap();
 
 			async move {
-				let uncles =
-					sc_consensus_uncles::create_uncles_inherent_data_provider(&*client, parent)?;
-
```

Comment:
No more uncles? Can you check if the polkadot repo -- the polkadot runtime -- also does not expect uncles anymore.

Reply:
Yup, it definitely seems like it - https://github.com/paritytech/substrate/blob/polkadot-v0.9.38/frame/authorship/src/lib.rs#L44

Reply:
Ah, lemme also check the runtime.

Reply:
No uncles mentioned here as well - https://github.com/paritytech/polkadot/blob/release-v0.9.38/runtime/polkadot/src/lib.rs

Reply:
Gotta dig deeper into the issue described here - https://github.com/paritytech/polkadot/blob/release-v0.9.37/runtime/polkadot/src/lib.rs#L340

Reply:
Interesting... I thought they were relevant for consensus, hence my question.